pork/smoked-pork-shoulder.json

```diff
@@ -0,0 +1,43 @@
+{
+    "title":"Smoked Pork Shoulder",
```

Comment:
I don't mind having multiple pork shoulder recipes. But this is the exact same name as the other one I just merged in. Can you change the name slightly? Also make sure to adjust the image name?

Reply:
Renamed both files